app/Http/Controllers/CountryController.php

```diff
@@ -3,13 +3,14 @@
 namespace App\Http\Controllers;
 
 use App\Models\Country;
+use App\Models\Team;
 
 class CountryController extends Controller
 {
     public function index()
     {
         // TASK: load the relationship average of team size
-        $countries = Country::all();
+        $countries = Country::withAvg(Team::class, 'size');
```

Comment:
This function doesn't take the model name as its first argument. It takes the function name defined for the relationship in the parent model. I.e `teams`. You can read more about it in the documentation for [aggregate functions](https://laravel.com/docs/8.x/eloquent-relationships#other-aggregate-functions).

```suggestion
        $countries = Country::withAvg('teams', 'size')->get();
```